jupyterhub_singleuser_profiles/ui/src/SizesForm/SizesForm.tsx

```diff
@@ -64,7 +64,7 @@ const SizesForm: React.FC<ImageFormProps> = ({ uiConfig, userConfig }) => {
   }, [sizeList]);
 
   React.useEffect(() => {
-    if (userConfig && sizeDescriptions) {
+    if (userConfig && sizeDescriptions.length > 0) {
```

Comment:
I think we should ensure `sizeDescriptions` is set before checking length as well.
```suggestion
    if (userConfig && sizeDescriptions?.length) {
```